lib/cmd/common/apps.js

```diff
@@ -47,7 +47,7 @@ function nguiApps(args, cb) {
     return doRead(projectId, appId, cb);
   }else if (action === 'create'){
     args.shift();
-    return create(args, cb);
```

Comment:
Yes, testing the cli needs a pattern defined for commands that will make api calls. Existing acceptance tests don't.
Lets get this fix published for now